tools/lib.sh

```diff
@@ -138,6 +138,9 @@ function build() {
   # Clean previous build artifacts.
   $MAKE clean
 
+  # Build doxygen and wiki docs
+  $MAKE docs
```

Comment:
If we remove this from the default build script I can create a job dedicated to `make docs` for cleaner/quicker output.